### DISCUSSION TITLE
UI crashes when going back through Fast Room Swiching

### DISCUSSION THREAD

**Discussion by leonardoazzi** [mozilla/hubs#5635] [Hubs Cloud Questions] [answered]
Hello my beloved community,

Since I merged the HC August update, I'm facing a crash with fast room switching. 
In my instance (merged with hubs-cloud), when I go to a room through a portal (fast room switching) and then I try to go to another one, the UI crashes. 
I know the feature is still experimental, but I could't reproduce the issue in hubs.mozilla.com (master). It works just as it should.
So, is there a fix to this issue not released to hubs-cloud yet?

![e7d80b13-a221-4502-a3d8-4caa5123ccaa](https://user-images.githubusercontent.com/13557537/182840582-60812d2e-9bb2-4e88-8a5d-7be7a85e43e5.png)

Thanks :)

**Comment by pattersonbl2** [comment, marked as answer]
This is a new issue in the hubs cloud.  I will convert this discussion into an issue.  due to the nature of it being an experimental feature, I am not sure how long before we have a fix.

**Reply by leonardoazzi** [reply]
Thanks, @Pattersonbl2! 
#5636
